src/land/smap_ssm2ioda.py

```diff
@@ -98,6 +102,8 @@ def _read(self):
             errs = errs[mask]
             qflg = qflg[mask]
             times = times[mask]
+            fill = (vals < valid_min) | (vals > valid_max) | (vals == _FillValue)
+            vals[fill] = float_missing_value
 
```

Comment:
```suggestion
        vals[vals == _FillValue] = float_missing_value

```

Reply:
@fabiolrdiniz think this is the way to do it will try to test it

Reply:
Oh... just saw this comment. Yes, similar to what I tried to say in my previous comment.